iocage/lib/Config/Jail/JailConfig.py

```diff
@@ -85,18 +85,18 @@ def _get_host_hostname(self) -> str:
                 return str(jail.humanreadable_name)
             raise e
 
-    def __setitem__(  # noqa: T484
+    def __setitem__(
         self,
         key: str,
         value: typing.Any,
-        **kwargs
```

Comment:
why was this necessary before?

Reply:
because of the `skip_on_error` property. But since that's the only argument it's better to specify it explicitly instead of `**kwargs`.